modyn/evaluator/internal/core_evaluation.py

```diff
@@ -0,0 +1,88 @@
+from collections.abc import Callable
```

Comment:
previously in `pytorch_evaluator`. The `pytorch_evaluator` itself is not suitable for usage in the `Performance-aware` policy. Therefore abstracting this code into it's own unit to avoid duplication